docker/Dockerfile

```diff
@@ -10,7 +10,8 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # gsutil is installed to deploy the book to Google Cloud Storage
-RUN pip2 install --upgrade pip && pip2 install gsutil
+RUN pip install --upgrade pip
+RUN pip install gsutil
```

Comment:
Are you sure this will work? `gsutil` is only available on Python 2 as far as I recall.

Reply:
Indeed, `gsutil` is only supported for py 2.7 however the build of the docker does work. The `Dockerfile` in master is not working for me.
I don't use Cloud Storage personally so I cannot confirm if everything will work as expected.

Do you have any commands I can run just to see if everything works as expected ?

Reply:
Apart from this one, all your fixes seem correct to me.  If you revert this one change, just it build for you?

Reply:
Unfortunately, it doesn't work, see the error I have when running `docker-compose build`.
I also tested with a full re-installation.
```
Traceback (most recent call last):
  File "/usr/bin/pip2", line 9, in <module>
    from pip import main
ImportError: cannot import name main
ERROR: Service 'book_builder' failed to build: The command '/bin/sh -c pip2 install --upgrade pip && pip2 install gsutil' returned a non-zero code: 1
```

Reply:
Ah, I think I see what might be happening.  Does this version of Ubuntu switch to Python 3 as a default Python?  If so, we may have to create both Python 2 and Python 3 virtual environments.